backend/route/contributors.route.js

```diff
@@ -0,0 +1,206 @@
+    //             const attendance = await Attendance.findOne({
+    //                 where: {
+    //                     DeveloperId: developer.id,
+    //                     ExperienceId: experience.id
+    //                 }
+    //             });
+    //             if (!attendance) {
+    //                 return Attendance.create({
+    //                     DeveloperId: developer.id,
+    //                     ExperienceId: experience.id
+    //                 });
+    //             }
+    //             return attendance;
+    //         } catch (e) {
+    //             return e;
+    //         }
+    //     }
+    // });
+
+    // // Remove association
+    // server.route({
+    //     method: 'DELETE',
+    //     path: '/api/v1/developer/{email}/experiences/{experienceId}',
+    //     options: {
+    //         validate: {
+    //             params: Joi.object({
+    //                 email: emailValidator.required(),
+    //                 experienceId: idValidator.required()
+    //             })
+    //         }
+    //     },
+    //     handler: async (request, h) => {
+    //         const { email, experienceId } = request.params;
+    //         try {
+    //             const developer = await getUser(email);
+    //             const attendance = await Attendance.findOne({
+    //                 where: {
+    //                     DeveloperId: developer.id,
+    //                     ExperienceId: experienceId
+    //                 }
+    //             });
+
+    //             if (!attendance) {
+    //                 throw Boom.notFound('This developer is not associated with that experience.');
+    //             } else {
+    //                 await Attendance.destroy({
+    //                     where: {
+    //                         DeveloperId: developer.id,
+    //                         ExperienceId: experienceId
+    //                     }
+    //                 });
+
+    //                 return { success: true };
+    //             }
+    //         } catch (e) {
+    //             return e;
+    //         }
+    //     }
+    // });
+
+    // // Edit association
+    // server.route({
+    //     method: ['PUT', 'PATCH'],
+    //     path: '/api/v1/developer/{email}/experiences/{experienceId}',
+    //     options: {
+    //         validate: {
+    //             payload: validateAttendanceUpdate,
+    //             params: Joi.object({
+    //                 email: emailValidator.required(),
+    //                 experienceId: idValidator.required()
+    //             })
+    //         }
+    //     },
+    //     handler: (request, h) => {
+    //         return getUser(request.params.email)
+    //             .then(developer => getAttendance(developer.id, request.params.experienceId))
+    //             .then(attendance => attendance.update(request.payload));
+    //     }
+    // });
+};
```

Comment:
Could you please remove the commented out code? Commented-out code should never be committed. [Google search](https://www.google.com/search?q=commented+out+code+should+not+be+committed&oq=commented+out+code+should+not+be+committed&aqs=chrome..69i57.4862j1j7&sourceid=chrome&ie=UTF-8).

Reply:
Sorry about that. Silly mistake.